account/accounts.go

```diff
@@ -44,11 +51,24 @@ func (e ErrCannotLocateKeyFile) Error() string {
 
 var zeroAddress = types.Address{}
 
+type SignParams struct {
+	Data     interface{} `json:"data"`
+	Address  string      `json:"account"`
+	Password string      `json:"password"`
```

Comment:
just curious is this a real password or just a password hash?

Reply:
This is a password hash